espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/viewaction/GetText.java

```diff
@@ -0,0 +1,50 @@
+package io.appium.espressoserver.lib.viewaction;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.support.test.espresso.ViewInteraction;
+import android.view.View;
+import android.widget.TextView;
+
+import org.hamcrest.Matcher;
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isDescendantOfA;
+import static android.support.test.espresso.matcher.ViewMatchers.isRoot;
+
+public class GetText {
+
+    private final View[] views = {null};
+
+    /**
+     * Hack way of getting a view. Similar to 'RootViewFinder' class.
+     */
+    private class GetTextViewAction implements ViewAction {
```

Comment:
This one is different from RootViewFinder.

RootViewFinder gets the root view

This one gets a view associated with a ViewInteraction.